wazuh/wazuh-kibana-app
# Replace generic HTTP status codes in the API controller

### Description
Hi team, this PR replaces HTTPS status error codes to prevent expected errors to be logged in the server console. 

Checking the connection of a downed API should now throw a 503 Service Unavailable.

![image](https://user-images.githubusercontent.com/9343732/196746440-b6c7a978-7453-4180-ac58-cd82aeb73362.png)


### Issues Resolved
https://github.com/wazuh/wazuh-kibana-app/issues/4650

### Test
- Create in `wazuh.yml` configuration a fake Wazuh API entry to cause a connection exception.
- Load the app
- Try to connect to that API
- Check for errors in the server logs

### Check List
